test/functional/sql-tag/sql-tag-parameters.ts

```diff
@@ -0,0 +1,5 @@
+// Tests have been split into driver-specific files:
+// - sql-tag-parameters-sqlite.ts
+// - sql-tag-parameters-better-sqlite3.ts
+// - sql-tag-parameters-mysql.ts
```

Comment:
Do we need this file? It's probably obvious enough how the tests are split.

If we keep it, ensure it references all the other files:

```suggestion
// - sql-tag-parameters-mysql.ts
// - sql-tag-parameters-postgres.ts
```

Reply:
that file was committed by mistake, sorry.